### DISCUSSION TITLE
Apakah Mecha CMS bisa menampilkan Total view dari keseluruhan halaman kang?

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#226] [Q&A]
Apakah Mecha CMS bisa menampilkan Total view dari keseluruhan halaman kang? kalau ane perhatikan view yang tampil hanya sesuai sesuai halaman saja

**Comment by taufik-nurrohman** [comment]
Bisa mas. Tapi kalau setiap kunjungan harus menghitung bakalan membebani situs web. Kecuali kalau total perhitungan disimpan ke dalam _cache_ yang dapat diperbarui misalnya 24 jam sekali.

~~~ php
$total = 0;

// v3.x.x
// foreach (g(LOT . D . 'page', 'data', true) as $k => $v) {
foreach (g(LOT . DS . 'page', 'data', true) as $k => $v) {
    if ('view.data' === basename($k)) {
        $total += (int) file_get_contents($k);
    }
}

echo $total;
~~~
